### PR TITLE
(UI) Indicate memory fragmentation

### LINKED ISSUE #374: Show warning instead of memory usage indicator with high value of quota_used_ratio
We got a situation where memory indicator is green but the memory allocation fails by reason of high memory fragmentation. 

See: https://www.tarantool.io/en/doc/1.10/book/box/box_slab/

### PR DESCRIPTION
**Pull request by iv-ov**
изображение

I didn't forget about

- [x] Tests
- [x] Changelog

Close #374

### CODE REVIEW

**Comment by iv-ov** [comment]
- [ ]  "WIP" status while waiting for new version of "@tarantool.io/ui-kit": "0.15.0" (2 new icons needed)

File: webui/src/misc/memoryStatistics.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  statistics: MemoryUsageRatios
+): FragmentationLevel => {
+  const arena_used_ratio = parsePercents(statistics.arena_used_ratio);
+  const quota_used_ratio = parsePercents(statistics.quota_used_ratio);
+  const items_used_ratio = parsePercents(statistics.items_used_ratio);
+
+  if (
+    arena_used_ratio > 0.9
+    && quota_used_ratio > 0.9
+    && items_used_ratio > 0.9
```
Comment (rosik) [inline webui/src/misc/memoryStatistics.js:27]:
```suggestion
    items_used_ratio > 0.9
    && arena_used_ratio > 0.9
    && quota_used_ratio > 0.9
```

NIT, but it's the right order of abstraction layers

File: webui/src/misc/memoryStatistics.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  ) {
+    return 'high';
+
+  } else if (
+    arena_used_ratio > 0.9
+    && quota_used_ratio > 0.6
+    && items_used_ratio > 0.6
```
Comment (rosik) [inline webui/src/misc/memoryStatistics.js:34]:
```suggestion
    items_used_ratio > 0.6
    && arena_used_ratio > 0.9
    && quota_used_ratio > 0.9
```

These constants differ from the issue, but they conform to tarantool doc and backend implementation

File: webui/src/components/ReplicasetServerListItem.js [hunk 3]
```diff
@@ -285,4 +285,27 @@ class ReplicasetServerListItem extends React.PureComponent<
...
+    switch (fragmentationLevel) {
+      case 'high':
+        return (
+          <Tooltip tag='span' content="Warning: Your memory is highly fragmented">
+            <IconChipDanger className={styles.iconMargin} />
+          </Tooltip>
+        );
+      case 'medium':
+        return (
+          <Tooltip tag='span' content="Warning: Your memory is fragmented">
```
Comment (rosik) [inline webui/src/components/ReplicasetServerListItem.js:301]:
```suggestion
          <Tooltip tag='span' content="Memory is highly fragmented">
```

File: webui/src/components/ReplicasetServerListItem.js [hunk 4]
```diff
@@ -285,4 +285,27 @@ class ReplicasetServerListItem extends React.PureComponent<
...
 }
 
+
+const MemoryIcon = (statistics: $PropertyType<Server, 'statistics'>) => {
+  if (statistics) {
+    const fragmentationLevel = getMemoryFragmentationLevel(statistics);
+    switch (fragmentationLevel) {
+      case 'high':
+        return (
+          <Tooltip tag='span' content="Warning: Your memory is highly fragmented">
```
Comment (rosik) [inline webui/src/components/ReplicasetServerListItem.js:295]:
```suggestion
          <Tooltip tag='span' content="Running out of memory">
```

**Pull request merged**